src/android/io/branch/BranchSDK.java

```diff
@@ -153,16 +153,24 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
                     this.getCreditHistory();
                     return true;
                 } else if (action.equals("createBranchUniversalObject")) {
-                    this.createBranchUniversalObject(args.getJSONObject(0));
+                    if (args.length() == 1) {
+                        this.createBranchUniversalObject(args.getJSONObject(0));
+                    }
```

Comment:
If the args length doesn't match, we should be returning false, otherwise the user will expect that their object is created and we wouldn't have created anything. same for the blocks of code below.